allennlp/service/cli/run.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+from contextlib import ExitStack
+import json
+import sys
+from typing import Sequence, Optional, IO
+
+from allennlp.service.servable import Servable, ServableCollection
+
+
+def parse_args(args: Sequence[str]):
+    parser = argparse.ArgumentParser(description="Run a model")  # pylint: disable=invalid-name
+    parser.add_argument('model', type=str, help='the name of the model to run')
+    parser.add_argument('input_file', metavar='input-file', type=str, help='path to input file')
+    parser.add_argument('--output-file', type=str, help='path to output file')
+    parser.add_argument('--print', action='store_true', help='print results to string')
+
+    return parser.parse_args(args)
+
+def get_model(args: argparse.Namespace) -> Optional[Servable]:
+    # TODO(joelgrus): use the args to instantiate the model
+    models = ServableCollection.default()
+    model_name = args.model
+    return models.get(model_name)
+
+def run(servable: Servable, input_file: IO, output_file: Optional[IO], print_to_console: bool) -> None:
+    for line in input_file:
+        data = json.loads(line)
+        result = servable.predict_json(data)
+        output = json.dumps(result)
+
+        if print_to_console:
```

Comment:
not really, `print(s, file=output_file)` feels unidiomatic to me, but I can't really justify why